experiment/simulation/js/main.js

```diff
@@ -1 +1,268 @@
 //Your JavaScript goes in here
+
+document.addEventListener('DOMContentLoaded', function(){
+
+	var play = true
+
+	var playButton = document.getElementById('play');
+	var pauseButton = document.getElementById('pause');
+	var restartButton = document.getElementById('restart');
+	var submitButton = document.getElementById('submit');
+
+	pauseButton.addEventListener('click', function() { window.clearTimeout(tmHandle) });
+	playButton.addEventListener('click', function() {  window.clearTimeout(tmHandle); tmHandle = setTimeout(draw, 1000 / fps); });
+	restartButton.addEventListener('click', function() {restart()})
+
+	function restart() 
+	{ 
+		window.clearTimeout(tmHandle) 
+
+		bldg = [
```

Comment:
Is bldg required to be a global variable? Please declare using const before use.

Reply:
Yes, bldg is global. It has been declared in line 130.